backend/notifications.py

```diff
@@ -40,10 +49,10 @@ def ExecuteNotifications(request, firebase):
     if notifications is None:
       return
     for notification_id, notification in notifications.iteritems():
-      if 'sent' not in notification and current_time > int(notification['sendTime']):
+      if notification['sendTime'] > 0 and current_time > int(notification['sendTime']):
```

Comment:
I was just being pedantic. Someone could alter the database directly with a negative value or something.